### ISSUE TITLE
Station rename

### ISSUE THREAD

**Issue by Bolsderac** [y20k/transistor#284]
Since 4.0, after station rename, station playing doesn't work. Always the same station is played whatever station clicked.

Otherwise, in Android auto, always the first station in list is played whatever the last played.

**Comment by y20k** [comment]
I will look into this. Hope to publish an update next weekend (I am currently afk).

**Comment by y20k** [comment]
I think I can start looking into that Tuesday evening. Could I - in the meantime - ask you to help me fixing the bug by giving me a detailed description of how to reproduce the problem? A step by step description beginning with the way you added the station that causes the problem would super-super helpful. Also App version and your Android version might help. 

I am sorry that the v4 update is such a mess at the moment. It took my by surprise. I had a beta version of the app running for month on my own phone and it was fine. But I guess my usage pattern is too narrow. I did not see all the problems coming.

**Comment by Bolsderac** [comment]
Don't worry.
i like this app i use to everyday in my car.

For the first issue, i can't reproduce it at every time i try to or the
issue is not the same each time.
I rename a radio (1A) and then i listen to another. When i return for
listening to this one i renamed, sometime, there is a time difference
between the radio listened to and the display (1B).
Other times, it's impossible to listen to another radio than this i
renamed. I have to delete all radio and to recreate each one.

For the second issue, i listen to a radio in transistor or throuth android
auto (2A). After Transistor processus is killed by android when i return in
android auto, the autostart of this one plays the first radio of the list,
not the last one it played (2B).

I'm not sure to really help you but it's better that nothing !

I use huawei p30 (ELE L29) on EMUI 10.0 (Android 10), Transistor 4.0.2.
Last version of Android Auto on Play Store.

Arnaud

**Comment by y20k** [comment]
@Bolsderac can you have a look at #285. (Let's discuss the Android Auto stuff in a separate issue.)

**Comment by y20k** [comment]
@Bolsderac 

Not sure, if it is important, but one question regarding the rename-issue: How to you add stations?

1. enter streaming URL in Transistor's search dialog
2. tap on streaming link in web-browser
3. built-in radio station search in Transistor's search dialog

**Comment by Bolsderac** [comment]
Choice 3

**Comment by jimishol** [comment]
I confirm that renaming do not allow to change stations. Trying to solve I discovered there is no delete station option. I uninstalled and installed hoping to create again the stations but it autodiscovered the previous ones. I could not find the station list in my storage so that I could remove it by hand. I would like the option to uninstall removing all relevant files or folders. Anyway this app is the only one that fits my need to hear my private mpd stream. So for its purpose it is the best. So I insisted by uninstalling and installing. The third or forth time it got the station right and started to function again.  Keep suuport flac, ogg and mp3. Dont mess with database stations. It is just fine. The only feature I would like to see is tag support but it is OK, I see them from mpd client. In this version I lost the station's icons. Any way thank you for the simplicity of your app. I am android 10 phone.

**Comment by y20k** [comment]
You should be able to delete a station, if you swipe the station off-screen

**Comment by jimishol** [comment]
Indeed!!! Swiping left and right gave me two more function with left being the delete one. 
Need to say that the problem may was the trantition to the new version. Now, the renaming works ok. Only the icons are not auto discovered. Forgot to mention earlier that the reason i prefer by far your application is that i do not need at all the hundreds of stations to choose from (that other apps do offer), it is the only one that allows me to input my own url and to actually work and mainly, why i do not use vlc or mpv? But your app has the lowest battery consumption of all.
So, keep it simple, congratulations and thank you.
